jsa/features/stanford_courses_tag_translate.feature

```diff
@@ -0,0 +1,96 @@
+  Scenario: Validate main admin page is there.
+    Given I am logged in as a user with the "administrator" role
+    And I am on "admin/stanford/courses/tag-translate"
+    Then I should see "New tag translation"
+
+  @api @content @dev
+  Scenario: Validate import admin page is there.
+    Given I am logged in as a user with the "administrator" role
+    And I am on "admin/config/stanford/courses/tag-translate/import"
+    Then I should see "Import" in the "Content" region
+
+  @api @content @dev
+  Scenario: Validate Export admin page is there.
+    Given I am logged in as a user with the "administrator" role
+    And I am on "admin/config/stanford/courses/tag-translate/export"
+    # No content on this page is permanent so it is impossible to check for some without a scenario breaking.
+
+  @api @content @dev
+  Scenario: Import content
+    Given I am logged in as a user with the "administrator" role
+    Given I am on "admin/config/stanford/courses/tag-translate/import"
+    Given I fill in "Import" with
+    """
+    {
+      "IR::intlsec": "Translation One",
+      "IR::wim": "This is Three",
+      "IR::core": "I am translation Two"
+    }
+    """
+    Then I press "Import"
+    Then I should be on "admin/config/stanford/courses/tag-translate"
+    And I should see "Import successful"
+    And I should see "IR::intlsec"
+    And I should see "IR::wim"
+    And I should see "IR::core"
+
+  @api @content @dev
+  Scenario: Create tag translations
+    Given I am logged in as a user with the "administrator" role
+    And I am on "admin/config/stanford/courses/tag-translate"
+    And I click "New tag translation"
+    Then I should be on "admin/config/stanford/courses/tag-translate/new"
+    Then I enter "ZZZZZ" for "edit-ctag"
+    Then I enter "zzzzzzzZZZZZzzzzzz" for "edit-chuman"
+    Then I press the "Save" button
+    Then I should see "Tag translation was saved successfully"
+
+  @api @content @dev
+  Scenario: Delete tag translations
+    Given I am logged in as a user with the "administrator" role
+    And I am on "admin/config/stanford/courses/tag-translate"
+    Then I click on the element with xpath "//table//tr[last()]//td[last()]//a[last()]"
+    Then I should be on "admin/config/stanford/courses/tag-translate/delete/4"
+    Then I should see "This action cannot be un-done. Choose wisely."
+    Then I press "Yes, delete this forever"
+    Then I should see "Tag translation was successfully removed."
+    Then I should be on "admin/config/stanford/courses/tag-translate"
+    Then I should not see "zzzzzzzZZZZZzzzzzz"
+
+  # This scenario only works if the import scenario about has been run
+  @api @content @dev @deploy
+  Scenario: Edit tag translations
+    Given I am logged in as a user with the "administrator" role
+    And I am on "admin/config/stanford/courses/tag-translate"
+    Then I click on the element with xpath "//table//a[1]"
+    Then I should be on "admin/config/stanford/courses/tag-translate/edit/1"
```

Comment:
I don't seem to be able to count... I thought I had that one nailed down. I am trying to ensure that only the items I want in the test are there. I wonder if their ids change.